### ISSUE TITLE
Could you add the Jupyter functionality to the extension?

### ISSUE THREAD

**Issue by ConAntares** [JuliaEditorSupport/julia-vscode#686]
The VScode extension Python has integrated Jupyter functionality. We can use #%% 
to run code in jupyter service. It is very  convenient. I hope that your extension can also add this functinality.

Sincerely,

**Comment by davidanthoff** [comment]
Isn't there a generic Jupyter extension that should in theory work with any language? I feel that is where this functionality would belong.

**Comment by runapp** [comment]
@davidanthoff Yes there was one extension named Jupyter. However according to its page it was merged into the Microsoft's Python extension.

**Comment by davidanthoff** [comment]
Hm, that is too bad... I saw a couple of other jupyter extensions on the marketplace, maybe one of them is taking that place?

**Comment by ConAntares** [comment]
> Hm, that is too bad... I saw a couple of other jupyter extensions on the marketplace, maybe one of them is taking that place?

Unfortunately, I couldn't find an extension can be used to Jupyter with complete functions. I have to use my Browser Firefox, that is not comfortable.
